lib/Completion/Bridge/TolerantParser/WorseReflection/WorseClassMemberCompletor.php

```diff
@@ -133,7 +134,8 @@ private function populateSuggestions(NodeContext $symbolContext, Type $type, boo
             return;
         }
 
-        foreach ($classReflection->methods() as $method) {
+        foreach ($type->members()->methods() as $method) {
```

Comment:
we enable ClassTypes to hold their own members() in addition to anything that might come from a reflected structure.

this might also provide support for anonymous classes.